docs/topics/generics.md

```diff
@@ -93,18 +97,15 @@ you don't get a `String`, but rather an `Object`.
 
 Joshua Bloch gives the name _Producers_ to objects you only _read from_ and _Consumers_ to those you only _write to_. He recommends:
 
->"For maximum flexibility, use wildcard types on input parameters that represent producers or consumers",
-> and proposes the following mnemonic:
->
->_PECS stands for Producer-Extends, Consumer-Super._
->
-{type="tip"}
+>"For maximum flexibility, use wildcard types on input parameters that represent producers or consumers,"
+
+and proposes the following mnemonic: _PECS_ stands for _Producer-Extends, Consumer-Super._
```

Comment:
```suggestion
>"For maximum flexibility, use wildcard types on input parameters that represent producers or consumers..."

He proposes the following mnemonic: _PECS_ stands for _Producer-Extends, Consumer-Super._
```
This formatting looks strange when you preview it so I've suggested to break it into two sentences.